tests/test_pykemon.py

```diff
@@ -8,207 +8,219 @@
 Tests for `pykemon` module.
 """
 
-import unittest
-import simplejson
+# import os, sys
+# sys.path.insert(0, os.path.dirname(os.getcwd()))
```

Comment:
Why are we doing this?

Reply:
It's just an artifact from my testing procedures.
I forgot to remove it.